src/java/org/apache/cassandra/repair/autorepair/AutoRepair.java

```diff
@@ -484,8 +498,8 @@ private void cleanupAndUpdateStats(RepairTurn turn, AutoRepairConfig.RepairType
                         TimeUnit.SECONDS.toDays(repairState.getClusterRepairTimeInSec()));
         }
         repairState.setLastRepairTime(timeFunc.get());
-
         repairState.setRepairInProgress(false);
```

Comment:
Should we also reset ToRepair/AlreadyRepaired metrics here when we are done with repairs?  Otherwise these metrics will be set to their completed values until the next repair, although that may be desirable, as it'll be more evident how much data was covered in the last repair, so maybe its better the way it currently is.